<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>spaceExband</class>
 <widget class="QWidget" name="spaceExband">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>81</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>文件名 ：</string>
   </property>
  </widget>
  <widget class="QLabel" name="filename_lab">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>30</y>
     <width>81</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>文件名</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>101</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>扩展类型 :</string>
   </property>
  </widget>
  <widget class="QComboBox" name="exband_type">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>80</y>
     <width>99</width>
     <height>24</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>扩展</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>缩小</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>111</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>扩展大小 :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="exband_size">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>130</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_exband">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>230</y>
     <width>112</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>130</y>
     <width>81</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>KB</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
